web/impact/impact/tests/test_organization_history_view.py

```diff
@@ -305,3 +308,13 @@ def test_options(self):
             results = response.data["actions"]["GET"]["properties"]["results"]
             get_options = results["item"]["properties"]
             assert_fields(OrganizationHistoryView.fields().keys(), get_options)
+
+    def test_options_against_get(self):
```

Comment:
Do you mean test_options vs test_options_against_get or do you mean all the occurrences of test_options_against_get?  Do we have existing examples of DRYing out tests in this way?  I would be happy DRY this out, but I want to do it in a way that we can adopt more generally since this is a common problem.  May be out of scope for this ticket.